### PR TITLE
Mh version beta 0.0.3

### PR DESCRIPTION
**Pull request by bleehu**
Cutting Dev (with bumped version number) to RC. Time to send our hard work to testing!

### CODE REVIEW

**Review by Turtlelord26 — COMMENTED** [review]
👍

File: 01_01_Basic_Rules.txt
Comment (1sourcecontrol) [inline 01_01_Basic_Rules.txt:63]: What's all this about "Rolling the Average"? 
I've never heard of this being used before. Is there a link to the thought behind this in github?

Anyways, rolling 2d10 divide by 2, i.e, (5+7)/2 = 6... 6%? This would be a worthless roll in a percentile system. I assume my interpretation can't be true. What does this mean?

File: 01_01_Basic_Rules.txt
Comment (1sourcecontrol) [inline 01_01_Basic_Rules.txt]: Head, shoulders, knees, and toes. Lol (line 179)

File: 01_01_Basic_Rules.txt
Comment (1sourcecontrol) [inline 01_01_Basic_Rules.txt]: another "<<<<<< HEAD"

File: 01_01_Basic_Rules.txt
Comment (1sourcecontrol) [inline 01_01_Basic_Rules.txt:1158]: These two paragraphs the way I originally wrote suppression. @bleehu after our phone conversation, I thought we were going to keep the footnotes? 
If that is the case, the two paragraphs after line 1098 need to be removed, and the asterisks that denote the reference to this footnote, need to be restored. I'm not sure what happened to the first footnote, because I wouldn't have used two asterisks unless there were two footnotes.

File: CharacterCreation/01_03_Races_Tundari
Comment (1sourcecontrol) [inline CharacterCreation/01_03_Races_Tundari]: This document is unplayable at the moment, as the fundamentals to the races rework are not written. Should this be left in DEV and not published to RC?

File: CharacterCreation/01_Races.txt
Comment (1sourcecontrol) [inline CharacterCreation/01_Races.txt:297]: Are "Siren Points" defined anywhere? I haven't seen them.

I consider several of the Opaleite subraces to be quite underpowered. But I'll let it go since the racial rework is in progress and I can bring up any concerns there.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (1sourcecontrol) [inline CharacterCreation/06_01_Systemized_Skills.txt]: I believe this should stay in DEV until complete -- and until it replaces the existing skills list. C2.0 will have issues with this current writing. Also there are lots of dev notes here, some of which digress from the direction of discussions I remember back in December. We should discuss this material -- lots of interesting things in here.

File: CharacterCreation/07_Items.txt
Comment (1sourcecontrol) [inline CharacterCreation/07_Items.txt:129]: Did I change these prices? I'm trying to remember who did.

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: These drones have not been completely converted to combat 2.0 with APLs or d10 accuracy.

**Review by 1sourcecontrol — CHANGES_REQUESTED** [review]
Some file concerns and lots of thoughts.

File: 01_01_Basic_Rules.txt
Comment (bleehu) [reply 01_01_Basic_Rules.txt:63]: @Turtlelord26 ?

File: 01_01_Basic_Rules.txt
Comment (bleehu) [reply 01_01_Basic_Rules.txt]: I'd call these blockers.

File: CharacterCreation/01_03_Races_Tundari
Comment (bleehu) [reply CharacterCreation/01_03_Races_Tundari]: I think you are correct; I'll filter it from the push.

File: CharacterCreation/07_Items.txt
Comment (bleehu) [reply CharacterCreation/07_Items.txt:129]: `$git blame`
looks like most of the prices are from Trowl223's commits back in the day, but Anfo pricing was me:
https://github.com/bleehu/Compound_X/commit/721e21199383764db768aa017eb1441867a8b7a6

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: blocker. Will change.

File: 01_01_Basic_Rules.txt
Comment (Turtlelord26) [reply 01_01_Basic_Rules.txt:63]: It first appeared in a Rev PR for racial trait purposes. I think at the time I read it as roll the check twice and average.

File: CharacterCreation/01_Races.txt
Comment (bleehu) [reply CharacterCreation/01_Races.txt:297]: The way I read it, I think Siren Points are Skill Points in a Siren Skill.

File: 01_01_Basic_Rules.txt
Comment (bleehu) [reply 01_01_Basic_Rules.txt:1158]: I did keep the footnotes? I modified the parser specifically to allow for multi-paragraph footnotes so we could keep these. 

As best I can tell from digging through the git history, there was a single one and then you changed it to two?: https://github.com/bleehu/Compound_X/commit/fc2e400052c2386c9e728cf63f10511411d0739d

Comment (1sourcecontrol) [reply 01_01_Basic_Rules.txt:1158]: I wonder if part of my post got cut-off somehow. Or I miswrote.
What I should've said was, there are two instances of the same writing; one as the original footnotes, and another as paragraphs. Lines 1096-1109 are the paragraphs.
https://github.com/bleehu/Compound_X/blob/2af435f3bf1976ba42eb9eb8b310104b2dde0a97/01_01_Basic_Rules.txt#L1096

Ahh, sorry for misplacing the blame (heh). Idk why I put those two asterisks in. Thanks for looking that up! And thanks for your hard work on the parser!

File: CharacterCreation/01_Races.txt
Comment (1sourcecontrol) [reply CharacterCreation/01_Races.txt:297]: I c

File: CharacterCreation/07_Items.txt
Comment (1sourcecontrol) [reply CharacterCreation/07_Items.txt:129]: Sounds good. I forgot I could look stuff up :P

File: 01_01_Basic_Rules.txt
Comment (1sourcecontrol) [reply 01_01_Basic_Rules.txt:63]: Is it actually being used for anything that anyone knows of? Just curious.

Also, it states that it prevents you from critically succeeding/failing, but that is certainly not true if the player rolls low enough or high enough. Unless the wording is supposed to mean that "Rolling the Average" grants those effects. In that case, I would like to see it stated more clearly.

Comment (1sourcecontrol) [reply 01_01_Basic_Rules.txt:63]: But this can be worked on another time. # nonblocking

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt
Comment (1sourcecontrol) [reply CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: When you fix this, I'll make sure to review again. (I see you asked for a 2nd review)

File: 01_01_Basic_Rules.txt
Comment (bleehu) [reply 01_01_Basic_Rules.txt:63]: Might I suggest picking this chat up in slack? https://compoundx.slack.com/messages/G8VFDBWET/

Comment (1sourcecontrol) [reply 01_01_Basic_Rules.txt:63]: I can't actually use slack on my laptop right now, slack won't load. So it's a bit of a pain.

**Comment by bleehu** [comment]
Requesting a re-review from @1sourcecontrol after merging some significant work on Engineer. Good enough for R_C?

File: 01_01_Basic_Rules.txt
Comment (1sourcecontrol) [reply 01_01_Basic_Rules.txt:1158]: --blocking

**Review by 1sourcecontrol — CHANGES_REQUESTED** [review]
I still need you to address line 1125 together with line 1160.
I tagged the comments with "--blocking"

These are the two instances of the (almost) exact same text. Is this duplication intentional?

After that, green lit

**Comment by bleehu** [comment]
R_C is a protected branch, I need an actual green light review before I merge.
(In an emergency, I can admin overwrite, but I'd prefer to save that for actual emergencies.)

**Comment by bleehu** [comment]
@Turtlelord26, I would admin merge this, but I'd much prefer that we get some consensus on this. The reviews should be pretty much done. 

This would be promoting the things that work in dev to RC. Work from as far back as August last year. Damn.

**Review by Turtlelord26 — APPROVED** [review]
Do This More Often.

Everything looks RC-able within the limits of my ability to review it all, with the possible exception of some minor points in systems that are already slated for heavy rework anyway.

**Pull request merged**
